### ISSUE TITLE
Reverse CCSPlayer::TransferInventory function

### ISSUE THREAD

**Issue by PiMoNFeeD** [SwagSoftware/Kisak-Strike#19]
its used when you take over a bot (and release control of a bot) to transfer equipment between the bot and yourself, leaked code contains only "remove what we have before doing anything" part and the rest is just missing

**Comment by LWSS** [comment]
Ahoy pimonfeed, thanks for reminding me of this. I have been somewhat busy lately but project isn't dead. Currently working on redo'ing hammer for linux.

**Comment by LWSS** [comment]
By the way, I ran into some people talking shit about you behind your back. Stay weary of the classic-offensive crowd, they seem salty for some reason.

**Comment by PiMoNFeeD** [comment]
> By the way, I ran into some people talking shit about you behind your back. Stay weary of the classic-offensive crowd, they seem salty for some reason.

ah I think I know what people you're talking about, they all got banned for their retardness and/or toxicity and now they're trying to prove everyone that (OPTIONAL)csco is better and my mod is shit because... because... because its using leaked code, yes, of course! (I have no idea how they compare a huge css mod with a ton of game- and engine code changes and csco which is pretty much just asset-replacing mod according to what zool told me himself)
they literally dont have any other facts against me lmao

**Comment by LWSS** [comment]
yeah it was a few months ago lol, I heard some of them whining publicly to Valve about an update breaking their mod or something and asked them why don't they just use the leaked source code instead of complaining. They went off on a tangent about how using the source code is evil and pimonfeed, yadda yadda out of nowhere lol. Honestly lost a lot of respect for them after that conversation and they haven't really done anything in the 4 years since it came out.

Reminds me of the dota modding schizos. They have a hate-boner for some guy named mocherad who actually makes mods instead of circle-jerking and being miserable in discord.

**Comment by PiMoNFeeD** [comment]
> yeah it was a few months ago lol, I heard some of them whining publicly to Valve about an update breaking their mod or something and asked them why don't they just use the leaked source code instead of complaining. They went off on a tangent about how using the source code is evil and pimonfeed, yadda yadda out of nowhere lol. Honestly lost a lot of respect for them after that conversation and they haven't really done anything in the 4 years since it came out.
> 
> Reminds me of the dota modding schizos. They have a hate-boner for some guy named mocherad who actually makes mods instead of circle-jerking and being miserable in discord.

exactly yeah, most people love living in a small toxic world of people fighting over who's vmt is made better for yet another very unique csgo models pack instead of actually learning to do something real, I personally literally learned how to work with player models (like re-rigging, fixing weight paints, other shit), with porting maps and lots of other small things literally from scratch, had no experience with any of that before making that mod of mine and yet the only thing people see is using leaked code **as if its a bad thing**

**Comment by PiMoNFeeD** [comment]
feel free to use this commit as a base: https://github.com/PiMoNFeeD/csgo-src/commit/aa823759ee516d19838d7cd5ff35d40bcac712bf
I don't think that I missed anything, but if I did - feel free to point me out lol
